Documentation Application Profiles and Workflow for Domain-specific Extensions (#543)
International-Data-Spaces-Association/InformationModel: evaluation_external/README.md

                           sh:maxCount 1;
                           sh:minCount 1;
                           sh:path dct:title;
                       ], [
                           sh:minCount 1;
                           sh:path dct:description;
                       ], [
                           sh:path dcat:keyword;
                           sh:minCount 0;
                           sh:maxCount 10;
                       ], [
                           sh:path dct:issued;
                           sh:datatype xsd:dateTimeStamp;
                           sh:minCount 1;
                           sh:maxCount 1;
                       ], [
                           sh:path ap15:serialNumber;
                           sh:minCount 1;
                           sh:maxCount 1;
                       ], [
                           sh:path ap15:operator;
                           sh:minCount 0;
                           sh:maxCount 1;
                       ] .

```

Now we see that the properties `dct:title`, `dct:description`, `dct:issued`, and `ap15@serialNumber` are mandatory, as the shapes indicates that the value for the `sh:minCount` is 1.

> Another possibility is to use the existent definitions in the **IDS Information model** and we want to tighten the restrictions, for example for the properties `dcat:mediaType`, `ids:representationStandard `, and `ids:instsance`.

Author: ```suggestion
Another possibility is to take existing definitions from the **IDS Information model** and tighten their restrictions, for example for the properties `dcat:mediaType`, `ids:representationStandard `, and `ids:instsance`.
```
